ntbs-service/Pages/LabResults/Index.cshtml.cs

```diff
@@ -174,20 +174,18 @@ private async Task PreparePageForGetAsync()
         private async Task FetchUnmatchedSpecimensAsync()
         {
             var permissionsFilter = await _userService.GetUserPermissionsFilterAsync(User);
+
             if (permissionsFilter.Type == UserType.NationalTeam)
             {
                 UnmatchedSpecimens = await _specimenService.GetAllUnmatchedSpecimensAsync();
+                return;
```

Comment:
It's probably a matter of personal preference, but I find
```
if (x)
{
    // do something
}
else
{
    // do something else
}
```
easier to read than
```
if (x)
{
    // do something
   return;
}
// do something else
```